setup.py

```diff
@@ -96,6 +96,7 @@ def get_version(version_file):
         "dnspython",
         "docopt",
         "mongoengine >= 0.27.0",
+        "pymongo >= 4.10",
```

Comment:
Since the pull request description says you want to add an upper bound and the change is a lower bound.

```suggestion
        "pymongo <= 4.10",
```

Reply:
🤦 Thanks for catching that!